libasn1fix/asn1fix_constr.c

```diff
@@ -60,7 +60,7 @@ asn1f_pull_components_of(arg_t *arg) {
 		coft = asn1p_expr_clone(terminal, 1 /* Skip extensions */);
 		if(!coft) return -1;	/* ENOMEM */
 
-		if(0) {
+		if(1) {
```

Comment:
No. Currently `asn1c` does not clash because I solved the clash root cause in scenarios I have seen.

For examples : 
1. If a constraint is of type `ACT_EL_TYPE`, the `asn1p_expr_t *` it referenced might be freed twice in `asn_expr_free()` and `asn1p_constraint_free()` then crash. 
I solved it in previous [pr 126](https://github.com/vlm/asn1c/pull/126/files) by introducing a `ref_cnt` field in `asn1p_expr_t`.
2. In `asn1p_y.y`, `$$->Identifier = "?"` cause clash when trying to free a constant character variable, it is solved by change it to `$$->Identifier = strdup("?");` in this pr.

Of course, I admit that perhaps there is still other asn.1 syntax combination will cause asn1c crash that I have encountered yet. But currently the test cases and various aforementioned asn.1 module examples are safe for this modification.

By the way, because `asn1p_y.c` won't be automatically generated for your repository, in addition to `asn1p_y.y`, I manually modify `asn1p_y.c` at corresponding code segment.